documentation-site/pages/components/layout-grid.mdx

```diff
@@ -172,6 +173,34 @@ Note how these are defined in a similar fashion to responsive props. Each value
 
 If you want to customize the breakpoints of the grid, you can do so via the `breakpoints` and `mediaQuery` properties. The grid will always pull breakpoints from the theme and correlate responsive props with those breakpoints.
 
+<Example title="Custom css units" path="layout-grid/custom-css-unit.js">
+  <CustomCssUnitExample />
+</Example>
+
+Layout grid uses `px` as a default css unit, you can change that by modifing theme in following manner:
+
+```js
+const theme = {
+  grid: {
+    unit: 'em',
+  },
+};
+```
+
+If you are using different css unit than pixle, you would also have to add `gutters` and `margins` in same css unit.For example if you are using `em` as a css unit `gutters` and `margins` might have value like below:
+
+```js
+const theme = {
+  grid: {
+    unit: 'em',
+    gutters: [1, 2, 2],
+    margins: [1, 2, 4],
+  },
+};
+```
+
+Grid component also accepts unit as a prop. By using this prop you can override theme's css unit for grid.
+
```

Comment:
Maybe add a snippet here showing the instance level setting of unit on Grid.

```js
<Grid gridUnit="em" gridMargins={[1, 2, 3]} gridMaxWidth={40}>{/* ... */}</Grid>
```

Reply:
Done 👍